apiary.apib

```diff
@@ -3650,6 +3650,17 @@ Add a draft work order.
             + type: `workOrder` (string)
             + id: `7c1d8672-f502-4333-9ea4-7a45add15115` (string)
 
+### workOrders.finalize [POST /workOrders.finalize]
+
+ Finalize a draft work order by generating the PDF.
```

Comment:
I'm not sure "by generating the PDF" is correct. Something like `which will generate the work order pdf` might make more sense?

Reply:
Just copied the description from the proposal. I like`which will generate ..` better, but will wait for some more input on this before changing, if you don't mind.

Reply:
Let's change this. Generating is a side effect, not the main thing of finalizing. The main thing is just the state of the WO that is being changed.